### DISCUSSION TITLE
Test webhook.site success but there is no error or ok msg in my site

### DISCUSSION THREAD

**Discussion by zrcshmily** [soranoo/TradingView-Free-Webhook-Alerts#90] [Q&A]
mysite: no ok or error msg
30-12-2024 17:31:03 | INFO      | Sending webhook alert<TradingView验证码>, content: 您的TradingView账号验证码是 696421
30-12-2024 17:31:03 | INFO      | The whole process taken 1.899476s
test site has ok msg
OK Sent webhook to https://webhook.site/xxxxxx successfu11y

**Comment by zrcshmily** [comment]
if res.status_code in [200, 201, 202, 203, 204]:
            log.ok(f"Sent webhook to {webhook_url} successfully, response code: {res.status_code}")
        elif retry_after := res.headers.get("Retry-After"):
            if res.status_code == 429:
                log.warning(f"Sent webhook to {webhook_url} failed, response code: {res.status_code}, Content: {payload}, Retry-After header({retry_after_header}) found, auto retry after {retry_after}s...")
                plan_to_run_run_at(time.time() + float(retry_after), send_webhook, payload)
                continue
                
            log.error(f"Sent webhook to {webhook_url} failed, response code: {res.status_code}, Content: {payload}.")

code is 403 but not defined in src

**Reply by soranoo** [reply]
The status code 403 is designed not to trigger retry because there is no way for an unauthorised request to succeed with retry.

**Reply by soranoo** [reply]
and based on the code snippet you provided, it is obvious the code you are using is outdated please pull the latest update if possible.
